test/access-token.test.js

```diff
@@ -108,6 +108,48 @@ describe('loopback.token(options)', function() {
       });
   });
 
+  it('should rewrite url for the current user literal at the end',
+    function(done) {
+      var app = createTestApp(this.token, done);
+      var id = this.token.id;
+      request(app)
+        .get('/users/me')
+        .set('authorization', id)
+        .end(function(err, res) {
+          assert(!err);
+          assert.deepEqual(res.body, {userId: '123'});
```

Comment:
You should not use magic values like `123` that are specified deep in test helpers. You should be able to read the user id from `token.userId`, right?

Please fix this issue in all other tests too.